src/util/options.rs

```diff
@@ -259,11 +335,11 @@ options! {
     // Should we eagerly finish sweeping at the start of a collection? (not supported)
     eager_complete_sweep:  bool                 [env_var: true, command_line: true]  [always_valid] = false,
     // Should we ignore GCs requested by the user (e.g. java.lang.System.gc)?
-    ignore_system_g_c:     bool                 [env_var: true, command_line: true]  [always_valid] = false,
-    // The upper bound of nursery size.
-    max_nursery:           usize                [env_var: true, command_line: true]  [|v: &usize| *v > 0 ] = DEFAULT_MAX_NURSERY,
-    // The lower bound of nusery size.
-    min_nursery:           usize                [env_var: true, command_line: true]  [|v: &usize| *v > 0 ] = DEFAULT_MIN_NURSERY,
+    ignore_system_gc:      bool                 [env_var: true, command_line: true]  [always_valid] = false,
```

Comment:
Our CI script currently uses this option -- I will update the script, and update the `ci-perf-kit` revision in the PR.

Reply:
Yes sorry. I was making the option more consistent with the others (like `full_heap_system_gc`).